src/routes/v2/authenticatedSites/contactUs.js

```diff
@@ -53,7 +53,7 @@ class ContactUsRouter {
     const router = express.Router({ mergeParams: true })
 
     router.get("/", attachReadRouteHandlerWrapper(this.readContactUs))
-    router.post("/", attachWriteRouteHandlerWrapper(this.updateContactUs))
+    router.post("/", attachRollbackRouteHandlerWrapper(this.updateContactUs))
```

Comment:
how do we choose bet using `attachWriteRouteHandlerWrapper` and `attachRollbackRouteHandlerWrapper`?

Reply:
attachWriteRouteHandlerWrapper -> single write only operation 

attachRollbackRouteHandlerWrapper -> multiple write operations 

reason for this change is becuase now iwth quickie, every "write" operation is two fold, and as such it follows the latter definition rather than the former